src/amo/components/Routes/index.js

```diff
@@ -78,6 +79,14 @@ const Routes = ({ _config = config }: Props = {}) => (
 
     <Route exact path="/:lang/:application/guides/:slug/" component={Guides} />
 
+    {_config.get('enableBlockPage') && (
+      <Route
+        exact
+        path="/:lang/:application/blocked/:guid/"
```

Comment:
I guess there's no regex somewhere that would potentially filter out some guids (I'm thinking something like the addons-server side default DRF regex that filtered out guids with `.` in them.)

Reply:
No, we don't have that. Our nginx config used to do this, though.